assets/dev/js/editor/elements/views/container.js

```diff
@@ -468,6 +488,76 @@ const ContainerView = BaseElementView.extend( {
 			this.$el.html5Droppable( this.getDroppableOptions() );
 		}
 	},
+
+	handleGridEmptyView() {
+		const currentContainer = this.getCorrectContainerElement();
+
+		this.moveElementToLastChild(
+			currentContainer,
+			currentContainer.find( '> .elementor-empty-view' ),
+		);
+	},
+
+	moveElementToLastChild( parentWrapperElement, childElementToMove ) {
+		let parent = parentWrapperElement.get( 0 ),
+			child = childElementToMove.get( 0 );
+
+		if ( ! parent || ! child ) {
+			return;
+		}
+
+		if ( parent.lastChild === child ) {
+			return;
+		}
+
+		parent.appendChild( child );
+	},
+
+	getCorrectContainerElement() {
+		return this.isBoxedWidth()
+			? this.$el.find( '> .e-con-inner' )
+			: this.$el;
+	},
+
+	shouldIncrementIndex( side ) {
+		if ( ! this.draggingOnBottomOrRightSide( side ) ) {
+			return false;
+		}
+
+		if ( this.isGridContainer() && this.emptyViewIsCurrentlyBeingDraggedOver() ) {
+			return false;
+		}
+
+		return true;
```

Comment:
Can it be shortened?
```suggestion
		return ! ( this.isGridContainer() && this.emptyViewIsCurrentlyBeingDraggedOver() );
```

Reply:
@udidol Done.